ebpf/scbpf/scbpf.c

```diff
@@ -0,0 +1,96 @@
+#include <linux/unistd.h>
```

Comment:
Should this have a copyright header? Also, coding style is different from Suricata's.

Reply:
Indeed.